.circleci/config.yml

```diff
@@ -275,10 +275,11 @@ commands:
           destination: generated-tests
 
 workflows:
-  nightly-coverage:
+  weekly-coverage:
     triggers:
       - schedule:
-          cron: "0 0 * * *"
+          # Saturday at 12:00 AM
+          cron: "0 0 * * 5"
```

Comment:
I couldn't find anything about which underlying cron implementation this is using, but 
ssn't 5 supposed to be Friday and 6 Saturday?

Reply:
Yes.